dbc_community.rb

```diff
@@ -0,0 +1,14 @@
+class DBC
+  attr_reader :age, :phase
+  attr_accessor :name
+
+  def initialize(options = {})
+    @phase = 1
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+  end
+
+  def offer_high_five
+    "High five!"
+  end
+end
```

Comment:
Are you referring to the age, phase and name of DBC? What could be a better name for this class?